package.json

```diff
@@ -1,19 +1,17 @@
 {
   "name": "add-variable-declarations",
+  "version": "0.0.0-development",
   "description": "Add variable declarations for global assignments.",
-  "main": "dist/add-variable-declarations.js",
-  "jsnext:main": "dist/add-variable-declarations.mjs",
+  "main": "src/index.js",
   "directories": {
     "test": "test"
   },
   "scripts": {
-    "build": "rollup -c",
     "test": "mocha",
-    "prepublish": "npm run build",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "prepublish": "tsc --project ."
```

Comment:
If I'm using `npm link` (or `yarn link`) to use this from another project, I'd still need to run `tsc --project .` each time to make changes accessible, right? IMO a `build` script is nice, although I may not be understanding the full details here.